### DISCUSSION TITLE
Use hotkey only for screen switching

### DISCUSSION THREAD

**Discussion by skydiverscott** [debauchee/barrier#1697] [Q&A]
I have setup a hot key to enable transitioning KM between computers screens.  Now I would like to disable the the screen to screen transition (based on the computer map layout) between computers leaving only hot-key to switch between computers.

How do I disable this transition method?

**Comment by skydiverscott** [comment]
Any suggestions on how to accomplish this goal?

**Reply by N2MBacon** [reply]
I too need this. When gaming in Fullscreen Borderless quick mouse movement will often trigger moving to Stream PC from Gaming PC

**Reply by N2MBacon** [reply]
Actually doing this helped. 
![image](https://user-images.githubusercontent.com/116976878/198849980-7fac40e0-6834-4a56-ab8e-653e751dfbd3.png)
![image](https://user-images.githubusercontent.com/116976878/198849988-612abef6-f913-4844-a0a7-a1920a86188a.png)
